tensorboard/plugins/debugger_v2/debugger_v2_plugin.py

```diff
@@ -55,3 +67,12 @@ def frontend_metadata(self):
         return base_plugin.FrontendMetadata(
             is_ng_component=True, tab_name="Debugger V2", disable_reload=True
         )
+
+    @wrappers.Request.application
+    def serve_runs(self, request):
+        runs = self._data_provider.list_runs(
+            debug_data_provider.DUMMY_DEBUGGER_EXPERIMENT_ID
```

Comment:
You're right. I didn't fully understand how `plugin_util.experiment_id()` worked. Now I see. It should work. I made changes to use it instead of the dummy experiment ID now.